os/board/rtl8730e/src/rtl8730e_mipi_lcdc.c

```diff
@@ -205,7 +207,23 @@ static void rtl8730e_control_backlight(uint8_t level)
 	float pwm_level = level/100.0;
 	lcdvdbg("level :%d , pwm level:%f\n", level, pwm_level);
 #if defined(CONFIG_LCD_ST7785) || defined(CONFIG_LCD_ST7701SN)
+#ifdef CONFIG_PM
+	if (level != 0 && !pm_lock_status) {	/* backlight ON, we should lock domain */
+		bsp_pm_domain_control(BSP_MIPI_DRV, 1);
+		pm_lock_status = 1;
+	} else if (level == 0 && pm_lock_status) { /* backlight OFF, we should release lock*/
+		bsp_pm_domain_control(BSP_MIPI_DRV, 0);
+		pm_lock_status = 0;
+	}
+#endif
```

Comment:
sorry for confusing, 
we plan to decision to enter the board sleep or not when the LCD is turned off will be made based on the os and driver's perspective.

Could you please remove here?
thanks

Reply:
Dear @ewoodev,
understood, so it will not be handled by rtk side, removed.
thanks